lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -899,7 +899,7 @@ public function getAlterTableSQL(TableDiff $diff)
     /**
      * @param \Doctrine\DBAL\Schema\TableDiff $diff
      *
-     * @return array|bool
+     * @return string[]|false
```

Comment:
As much as I hate using `false` as type, there is no better way around here. Usually these cases are something to replace with `null` for 3.x.